practice-app/website/api/event.py

```diff
@@ -78,3 +78,73 @@ def event():
             return "Try Later", 403
         
         return jsonify(new_event.serialize()), 201
+
+
+
+
+
+# When the id of the event given, corresponding discussion is returned by adding the definition of the sport type in the json format
+# Corresponding event must exist and have a sport type in db.
+@events.route('<event_id>/discussions', methods=['GET', 'POST'])
```

Comment:
If there is not a post option maybe removing it from the methods could be better.